Library/Homebrew/os/mac/xcode.rb

```diff
@@ -304,7 +304,9 @@ def update_instructions
             sudo xcode-select --install
 
           Alternatively, manually download them from:
-            #{Formatter.url("https://developer.apple.com/download/more/")}.
+            #{Formatter.url("https://developer.apple.com/download/more/")}
+          after checking the latest available version for your system on:
+            #{Formatter.url("https://developer.apple.com/support/xcode/#minimum-requirements")}.
```

Comment:
We know these versions so we could perhaps just tell people the versions instead?

Reply:
see `latest_version` or `minimum_version`

Reply:
Are they the same as the XCode versions, or can the version numbers be different across the two? I assumed they are disconnected, since apparently I had the latest XCode but not the latest CLT version installed

Reply:
The versions differ, right. @Bo98 any thoughts here (as you've been thinking about this recently)

Reply:
To better explain my point: latest_clang_version is what is used to determine whether the CLTs are outdated. The number in 
https://github.com/Homebrew/brew/blob/1810ff9722c05cedf3cf9e20670f0a26da519e8c/Library/Homebrew/os/mac/xcode.rb#L349-L354

But these are based on 
https://github.com/Homebrew/brew/blob/1810ff9722c05cedf3cf9e20670f0a26da519e8c/Library/Homebrew/os/mac/xcode.rb#L314-L320

which is not the kind of versioning used by Apple on the website where you download the CLTs.

Reply:
Actually, I'm not sure anymore whether there is a problem in practice.

The goal of the message is to let the user know which version is the latest supported CLT version on their OS, so it only has to match the version that is found on Apple's website, right? Which seems consistent with the latest XCode's version for the same system.
I'm not sure *why* my tools were outdated while my XCode was not, and how I installed them. However, the latest CLT on Apple's website is still 11.3.1 - same as latest XCode for Mojave - and that's what I downloaded and installed to fix the problem.

So maybe it does make sense to indicate XCode's version to the user, not just to check for it to determine whether the tools are up-to-date?

Reply:
> So maybe it does make sense to indicate XCode's version to the user, not just to check for it to determine whether the tools are up-to-date?

Yes, this makes sense to me 👍🏻 `MacOS::Xcode.latest_version` will get this version for you.

Reply:
alright, if the wording is ok (not a native speaker) this should be it